packages/nimble-components/src/rich-text-editor/tests/rich-text-editor.stories.ts

```diff
@@ -50,15 +59,24 @@ Note: The content in the \`footer-actions\` slot will not adjust based on the st
 client application to make any necessary adjustments. For example, if the buttons should be disabled when the rich-text-editor is disabled, the
 client application must implement that functionality.
 `;
+const fitToContentDescription = `Setting \`fit-to-content\` allows the editor to grow vertically to fit the content instead of displaying a
+vertical scrollbar when content exceeds the visible height.
+
+However, \`fit-to-content\` will not take effect if the \`max-height\` of the element is set to the element i.e. vertical scrollbar will be enabled if it
+exceeds the specified maximum height.`;
```

Comment:
nit: wording suggestion
```suggestion
When \`fit-to-content\` is set the component will only grow to its \`max-height\`. A vertical scrollbar will be enabled when the content it exceeds the specified maximum height.`;
```

Reply:
I have updated as suggested but the `fit-to-content` will be moved to a different PR.